README.md

```diff
@@ -12,44 +12,82 @@ TBD
 
 ### Environment Setup
 
-#### Fedora
+In order to develop the project you need to install following dependencies.
+
+#### Prerequisities
+
+To build the project on CentOS Stream 9 you need to enable CodeReady Build repository:
 ```bash
-sudo dnf install clang-tools-extra gcc make systemd-devel
+sudo dnf install -y dnf-plugin-config-manager
+sudo dnf config-manager -y --set-enabled crb
+```
+
+#### Dependencies installation
+
+```bash
+sudo dnf install -y clang-tools-extra gcc meson systemd-devel
 ```
 
 ### Code Style
 
-`clang-format` is used to enforce a unified code style. For setup instructions, please refer to the [official Clang getting started guide](https://clang.llvm.org/get_started.html). 
+`clang-format` is used to enforce a unified code style. For setup instructions, please refer to the [official Clang getting started guide](https://clang.llvm.org/get_started.html). `clang-format` is integrated into the
+[meson](https://mesonbuild.com/) build system.
 
 Once `clang-format` is set up, all source files can be formatted via:
 ```bash
-make fmt
+meson setup builddir
```

Comment:
If we stick with the name `builddir`, it should be added to `.gitignore` and the existing `bin` can be removed.

Reply:
Good idea!

Reply:
Done